rpcs3/Emu/RSX/VK/VKPipelineCompiler.cpp

```diff
@@ -32,6 +32,10 @@ namespace vk
 
 	void pipe_compiler::operator()()
 	{
+#ifndef _WIN32
+		// Set low priority
+		thread_ctrl::scoped_priority low_prio(-1);
+#endif

```

Comment:
Are you sure these threads aren't used ingame?
Why decrease the priority?

Reply:
These are the asynchronous compile threads, they're set to lower priority so that they don't interfere with game threads.

Reply:
they are also used ingame

Reply:
remnants from personal testing. removed.